sql/sqlfmt.go

```diff
@@ -38,8 +38,14 @@ func GenerateCreateTableStatement(tblName string, colStmts []string, tblCharsetN
 
 // GenerateCreateTableColumnDefinition returns column definition string for 'CREATE TABLE' statement for given column.
 // This part comes first in the 'CREATE TABLE' statement.
-func GenerateCreateTableColumnDefinition(col *Column, colDefault string) string {
-	stmt := fmt.Sprintf("  %s %s", QuoteIdentifier(col.Name), col.Type.String())
+func GenerateCreateTableColumnDefinition(col *Column, colDefault string, tableCollation CollationID) string {
```

Comment:
we currently use this in Dolt as a shortcut for generating a create table statement, but dolt bump will need an edit